llama_stack/providers/inline/safety/llama_guard/llama_guard.py

```
@@ -189,6 +216,44 @@ async def run_shield(
 
         return await impl.run(messages)
 
+    async def run_moderation(
+        self,
+        input: str | list[str],
+        model: str | None = None,  # To replace with default model for llama-guard
+    ) -> ModerationObject:
+        if model is None:
+            raise ValueError("Model cannot be None")
+        if not input or len(input) == 0:
```

I don't think we should raise here, just silently return safe.